views/common/head.ejs

```diff
@@ -16,6 +16,8 @@
       href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css"/>
 
 
+<link rel="stylesheet" href="/assets/fa/css/dvna-dep-styling.css">
```

Comment:
NITPICK, NONBLOCKING: The Add Salt used to show up like a horizontal switch. Did you intend to change the look to a traditional checkbox? If not, I think the stylesheets are interacting so now the switch input from bootstrap 5 isn't working. 
Here's what it looks like now:
![image](https://user-images.githubusercontent.com/12928553/128201914-927d0bbb-72e6-4854-b409-0b608588f88c.png)

Here's what it should look like with bootstrap 5:
![image](https://user-images.githubusercontent.com/12928553/128202286-29eeb7bf-37a8-4089-8a16-024d2bcd88b4.png)

Reply:
No worries. -- I just looked into this, and think there's a misunderstanding.
 
In your last pull request, I actually noticed that you commented out bootstrap 5 link in head.ejs.  So when I pulled it for my review, I never noticed the salt feature appearing as a switch (it was always a checkbox for me).  I assume that you did it done on purpose because un-commenting bootstrap 5 in head.ejs seem to cause more problems.  Mainly, it seems to mess with other DVNA UI because it was done in bootstrap 3.3.7.  In fact, seeing that it's too much work to update all legacy UI to bootstrap 5, I just did all my UI work using bootstrap 3.3.7.

That said, even if bootstrap 5 was present, your switch should not be affected by the dvna-dep-styling.css, which only targets things with id named "vuln_subheader" or "sc_indicator1" (your salt feature id is "saltSwitch")

Hope that helps.

![image](https://user-images.githubusercontent.com/47507778/128209036-80a8a90a-f210-46ba-8285-650a92477130.png)